### ISSUE TITLE
[Enhancement] next in series feature

### ISSUE THREAD

**Issue by raviwarrier** [advplyr/audiobookshelf#1591]
### Describe the feature/enhancement

It would be nice if there next book in series was populated in the list when the current book reaches 90-95%. This would allow people to start downloading the book in parallel and start listening to the next book as soon as the current one is over (for those who binge-listen).

Additionally, one could have a prompt (toggle in settings) that asks the user if they want to start downloading the next book at the same 90-95% mark.

**Comment by advplyr** [comment]
Or are you talking about downloading on the mobile app and this is in the wrong repo?

**Comment by raviwarrier** [comment]
> Or are you talking about downloading on the mobile app and this is in the wrong repo?

Yes, the second part of the enhancement was for the app. But I thought since the new series list is populated on the server side, I posted it here. Ignore the second part.

I'll explain better (hopefully).

Say I am listening to a book. I am reaching the end (book progress is 90%). It sends a progress sync to the server (if connected) and the server then adds the next book to "Continue Series" list. Right now, it populates once the book is complete. The reason I asked for this enhancement is because I was listening to a book and wanted to download the next book before I finished it (I had about 30m left in the book). But since it wasn't there in the "Continue Series" list, I had to go to the series page, scroll, open the series, open the book and then click download. By including the enhancement, you can save the user the first 3 steps.
